tests/test_auto_parallel/test_tensor_shard/test_modeling_clip.py

```diff
@@ -0,0 +1,33 @@
+import sys,os
+import torch
+import torch.nn as nn
+from torch.fx import symbolic_trace
+from torch.fx import GraphModule
+from PIL import Image
+import requests
+from transformers import CLIPProcessor, CLIPModel
+from torch.fx import symbolic_trace
+from colossalai.fx.tracer import ColoTracer
+
+model = CLIPModel.from_pretrained("openai/clip-vit-base-patch32")
+model = model.to('meta')
+processor = CLIPProcessor.from_pretrained("openai/clip-vit-base-patch32")
```

Comment:
don't load pretrained weights, cuz for the CI testing it will spend a lot of time.
Can refer to these files.
https://github.com/hpcaitech/ColossalAI/tree/main/tests/test_fx/test_tracer/test_hf_model

Reply:
`from_pretrained` is still not removed, @tangbogreat can you help to remove this to save time for unit testing?